spec/index.html

```diff
@@ -2184,7 +2214,9 @@ <h5>The <code>&lt;<dfn id="the-properties-element">properties</dfn>&gt;</code> e
         </dl>
 
         <p>
-          A <a href="#the-feature-element"><code>feature</code></a> element can have zero or one <a href="#the-properties-element"><code>properties</code></a> element, which contains zero or more unknown elements, whose content is text.
+          A <a href="#the-feature-element"><code>feature</code></a> element can have zero or one 
+          <a href="#the-properties-element"><code>properties</code></a> element, 
+          which contains some <a href="https://html.spec.whatwg.org/multipage/dom.html#palpable-content">palpable content</a>.
```

Comment:
This is a paragraph for the `properties` element but it's partially describing the `feature` element's content model, and its own content model which already is already defined in the description list. I think it's safe to remove this paragraph altogether.

Reply:
Thanks, hopefully resolved. Deleting redundant spec text is a joy.